README.md

```diff
@@ -80,100 +71,95 @@ Now we'll convert it using HTML5 Semantic Elements
+As a developer, keep in mind that while these elements are intended for certain content, there are no hard rules about how to configure them. You should feel comfortable configuring them in any way that makes the most sense to you and best suits your layout needs.
 
-#### Article
+The following descriptions come from the documentation at the Mozilla Developer Network.
+
+### Article
 
 `<article>...</article>`
 
 The HTML `<article>` element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication). This could be a forum post, a magazine or newspaper article, a blog entry, an object, or any other independent item of content. Each `<article>` should be identified, typically by including a heading (`<h1>-<h6>` element) as a child of the `<article>` element.
 
-#### Aside
+### Aside
 
 `<aside>...</aside>`
 
 The HTML `<aside>` element represents a section of the page with content connected tangentially to the rest, which could be considered separate from that content. These sections are often represented as sidebars or inserts. They often contain the definitions on the sidebars, such as definitions from the glossary; there may also be other types of information, such as related advertisements; the biography of the author; web applications; profile information or related links on the blog.
 
-#### Details
+### Details
 
 `<details>..</details>`
 
 The HTML Details Element (`<details>`) is used as a disclosure widget from which the user can retrieve additional information.
 
-#### Figure & Figcaption
+### Figure & Figcaption
 
 `<figure>...<figcaption>...</figcaption></figure>`
 
 The HTML `<figure>` element represents self-contained content, frequently with a caption (`<figcaption>`), and is typically referenced as a single unit. While it is related to the main flow, its position is independent of the main flow. Usually this is an image, an illustration, a diagram, a code snippet, or a schema that is referenced in the main text, but that can be moved to another page or to an appendix without affecting the main flow.
 
 The HTML `<figcaption>` element represents a caption or a legend associated with a figure or an illustration described by the rest of the data of the `<figure>` element which is its immediate ancestor which means `<figcaption>` can be the first or last element inside a `<figure>` block. Also, the HTML Figcaption Element is optional; if not provided, then the parent figure element will have no caption.
 
-#### Footer
+### Footer
 
```

Comment:
I think HTML would work nicely for footer and header sections. It might be kind of cool to pull the structure of the header/footer from Learn, so they can see it? I'm not sure how complicated they are though - we might need to simplify it.